pop/service/struct.go

```diff
@@ -0,0 +1,74 @@
+package service
+
+/*
+This holds the messages used to communicate with the service over the network.
+*/
+
+import (
+	"gopkg.in/dedis/crypto.v0/abstract"
+	"gopkg.in/dedis/onet.v1/network"
+)
+
+// We need to register all messages so the network knows how to handle them.
+func init() {
+	for _, msg := range []interface{}{
+		CheckConfig{}, CheckConfigReply{},
+		PinRequest{},
+	} {
+		network.RegisterMessage(msg)
+	}
+}
+
+// CheckConfig asks whether the pop-config and the attendees are available.
+type CheckConfig struct {
+	PopHash   []byte
+	Attendees []abstract.Point
+}
+
+// CheckConfigReply sends back an integer for the Pop:
+// - 0 - no popconfig yet
+// - 1 - popconfig, but other hash
+// - 2 - popconfig with the same hash but no attendees in common
+// - 3 - popconfig with same hash and at least one attendee in common
```

Comment:
Do you use these differents variable somewhere? I mean do you have a 
```go
switch reply.PopStatus {
  case 1:
  case 2:
  ...
}
```
If so, it'd be nice to put constant values for those then. like `PopConfigAbsent` and `PopConfigDiffentHash` `PopConfigNoCommonAttendee` or something like that to avoid comparing hardcoded values.